src/project/activator.ts

```diff
@@ -60,4 +61,7 @@ export function activate(ballerinaExtInstance: BallerinaExtension) {
 
     // activate the create kubernetes.toml command
     activateK8sCommand();
+
+    // activate the add module command
+    activateAddCommand();
```

Comment:
May be we can rename to `activateAddModuleCommand`

Reply:
We've been using Ballerina CLI command names for plugin command names. Will try to improve later.